CHANGELOG.md

```diff
@@ -70,10 +70,11 @@ Improvements:
 		- Smarter restoration of the previous scalar fields loading pattern
 		- Maximum GPS time shift increased to 10^10
 	- LAS file saving dialog
-		- CC will now automatically assign scalar fields with non 'LAS-standard' names to Extra fields (VLRs)
-		- if the 'Save remaining scalar fields as Extra fields / VLRs' checkbox is checked (default state),
-			some entries are automatically created in the 3rd tab (Extra fields / VLRs). This is updated automatically
+		- CC will now automatically assign scalar fields with non 'LAS-standard' names to Extra fields (EVLRs)
+		- if the 'Save remaining scalar fields as Extra fields / EVLRs' checkbox is checked (default state),
+			some entries are automatically created in the 3rd tab (Extra fields / EVLRs). This is updated automatically
```

Comment:
We have to keep the `VLR` name here as `EVLR`  in LAS context generally refers to `Extended Variable Lenght Record`  and ExtraBytes VLRs are stored in VLRS not Extended VLRS

Reply:
or use EB-VLR as the name to not have the ambiguity

Reply:
Yes, I see your point 😅